### ISSUE TITLE
Unicast

### ISSUE THREAD

**Issue by Marte2007** [duncanthrax/scream#25]
Is it possible to have a configurable unicast mode, instead of multicast?
I'm asking this for two reasons: reducing network traffic (with switches without IGMP snooping) and allowing more than one sender-receiver pair in the same network without interferences.

Maybe you can read the target address from the registry?
https://docs.microsoft.com/en-us/windows-hardware/drivers/kernel/registry-key-object-routines

**Comment by duncanthrax** [comment]
Yep that looks good. I'll take a look.

**Comment by martinellimarco** [comment]
I second this request, it may solve routing problems too.
At the moment I have a VM with Scream and 2 network interfaces.
I was not able to hear anything at first, I inspected the network with wireshark and I noticed that the packets were sent only on one interface, the one connected to a subnet where I wasn't running a receiver.
I disabled that interface and the packets went to the other one, and I was able to hear the sound on the receiver.
With this I can just set the right destination and the routing mechanism will do the rest.
